src/pages/waiting-room/components/cbt-number/cbt-number.ts

```diff
@@ -19,12 +20,22 @@ export class CBTNumberComponent implements OnChanges {
 
   formControl: FormControl;
   static readonly fieldName: string = 'cbtNumber';
+  readonly dl196cbtCertNumberValidator = getDL196CBTCertificateNumberValidator();
 
   ngOnChanges(): void {
     if (!this.formControl) {
       this.formControl = new FormControl(null);
       this.formGroup.addControl(CBTNumberComponent.fieldName, this.formControl);
     }
+
+    if (this.cbtNumber.length === +this.dl196cbtCertNumberValidator.maxLength) {
+      this.formGroup.get(CBTNumberComponent.fieldName).clearValidators();
```

Comment:
Why do we clear validators?
Surely we should always be enforcing the fact it is required and should follow a specified format

Reply:
i had it originally without clearing the validators and it refused to allow leaving the page when another item was changed (another box clicked).  This was my work around.

Reply:
Surely if no pattern is enforced it could be complete gibberish but the correct length?

Reply:
happy to let you look at it.

Reply:
Have looked at this again.  If i leave the validator in force im getting really inconsistant behaviour when i click away from the field.  sometimes the same number will be valid and sometimes not.  By clearing the validator it will not allow complete gibberish as only numerics are allowed to be entered into this field.